common/pjpeg.c

```diff
@@ -833,6 +833,12 @@ pjpeg_t *pjpeg_create_from_file(const char *path, uint32_t flags, int *error)
     uint8_t *buf = malloc(buflen);
     fseek(f, 0, SEEK_SET);
     int res = fread(buf, 1, buflen, f);
+
+    int error_code = ferror(f);
+    if (error_code ){
+        printf ("Read failed with error code: %d\n", error_code );
+    }
```

Comment:
As mentioned before, do not print from the library in release mode. Use the debug print or errno to indicate errors.